wmn-data.json

```diff
@@ -1201,6 +1201,22 @@
         "known" : ["jaxjames", "jordan-2"],
         "cat" : "xx NSFW xx"
        },
+       {
+        "name": "coub",
+        "uri_check": "https://coub.com/{account}",
+        "e_code": 200,
+        "e_string": "- Channel on Coub</title>",
+        "m_string": "Sorry, that page doesn't exist.",
+        "m_code": 404,
+        "known": [

```

Comment:
While this is valid JSON format, please collapse this and put all `known` on a single line like the other entries have it.